website/generate_excel_markdown.py

```diff
@@ -248,10 +254,11 @@ def get_p_value(cmd, p_name) -> str:
         standard_eg = standard_eg.strip(f"{sep} ") + ")"
 
         if required_eg == standard_eg:
-            return {"A. Required": required_eg}
-        # Uncomment to add standard examples
-        # return {"A. Required": required_eg, "B. Standard": standard_eg}
-        return {"A. Required": required_eg}
+            return {"Required": required_eg}
+
+        if cmd in ("/get", "/byod"):
```

Comment:
Don't fully understand what this does

Reply:
We can remove the if statement for now. Need to refactor the code to use platform examples, waiting new version merge.